### DISCUSSION TITLE
Is it possible to selectively disable `navigation.expand` for a single section?

### DISCUSSION THREAD

**Discussion by johnthagen** [squidfunk/mkdocs-material#5707] [Q&A] [answered]
In our documentation we want to expand the navigation for most sections, but selectively hide it for a few less important sections.

Is this possible?

**Comment by squidfunk** [comment, marked as answer]
Duplicate of #2185.

**Reply by johnthagen** [reply]
Thank you for the information. Sorry for the duplicate question.

**Reply by squidfunk** [reply]
No worries. We'll investigate in the future what features might be possible to scope to pages or sections of the documentation, and `navigation.expand` is an interesting candidate. However, we can't currently put an estimate on when this will be available.
